src/v/raft/probe.cc

```diff
@@ -146,6 +146,24 @@ void probe::setup_metrics(const model::ntp& ntp) {
          [this] { return _recovery_request_error; },
          sm::description("Number of failed recovery requests"),
          labels)
+         .aggregate(aggregate_labels),
+       sm::make_counter(
+         "recovery_requests",
+         [this] { return _recovery_requests; },
+         sm::description("Number of recovery requests"),
+         labels)
+         .aggregate(aggregate_labels),
+       sm::make_counter(
+         "group_configuration_updates",
+         [this] { return _configuration_updates; },
+         sm::description("Number of raft group configuration updates"),
```

Comment:
I'm not entirely sure what this intends to track. Michal, can you confirm please?

Reply:
it ios indeed the number of configuration changes